tests/vm

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eux
+
+# Install LXD
+install_lxd
+
+# Configure LXD
+lxd init --auto
+
+IMAGE="${TEST_IMG:-ubuntu-minimal-daily:24.04}"
+
+# Launch test instance
+lxc launch "${IMAGE}" vm1 --vm
+waitInstanceBooted vm1
+
+echo "==> Test cleanly stopping a VM"
+lxc stop vm1
+lxc start vm1
+waitInstanceBooted vm1
```

Comment:
I think this could be moved inside the `if` as there's no point in starting the VM back if we are to delete it right after it's done booting.